news/2022-07-27-new-featured-artist-pegboard-nerds.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+title: "New Featured Artist: Pegboard Nerds"
+date: 2022-07-27 10:00:00 +0000
+---
+
+Welcome aboard our latest Featured Artist, **Pegboard Nerds**! Because we know you can tell talent when you see it.
+
+![](https://assets.ppy.sh/artists/287/header.jpg)
+
+Among [Monstercat](https://osu.ppy.sh/beatmaps/artists/255)'s flagship artists, **Pegboard Nerds** is a Scandinavian duo comprised of two veteran electronic producers. While both members have been constructing tunes since the '90s, they only joined forces in 2011 and have been transforming the EDM world for the last decade.
+
+You'll come across **11** breathtaking tracks when you see [**Pegboard Nerds**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/287), all of which are pre-timed and ready for mapping!
```

Comment:
```suggestion
You'll come across **11** breathtaking tracks when you visit [**Pegboard Nerds**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/287), all of which are pre-timed and ready for mapping!
```

Reply:
pretty sure this is for the meme value yeah, will italicise